examples/example_annubes.ipynb

```diff
@@ -60,18 +116,59 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "env = AnnubesEnv()\n",
-    "env = DummyVecEnv([lambda: env])\n",
-    "# plot example trials with random agent\n",
-    "data = ngym.utils.plot_env(env, num_trials=5, ob_traces=[\"fixation\", \"start\", \"v\", \"a\"], model=model)"
+    "env1 = AnnubesEnv({\"v\": 1, \"a\": 0})\n",
+    "env1_vec = DummyVecEnv([lambda: env1])\n",
+    "model = A2C(\"MlpPolicy\", env1_vec, verbose=0)\n",
+    "model.learn(total_timesteps=10000)\n",
+    "env1_vec.close()\n",
+    "\n",
+    "# plot example trials with trained agent\n",
+    "data = ngym.utils.plot_env(env1, num_trials=10, ob_traces=[\"fixation\", \"start\", \"v\", \"a\"], model=model)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# switch to the second environment and continue training\n",
+    "env2 = AnnubesEnv({\"v\": 0, \"a\": 1})\n",
+    "env2_vec = DummyVecEnv([lambda: env2])\n",
+    "model.set_env(env2_vec)\n",
+    "model.learn(total_timesteps=10000)\n",
```

Comment:
I think it would be nice if, before learning the new task, we already do a plot, showing (and commenting to highlight i) that at this point the model cannot resolve the auditive task yet. This will then contrast with the "end point" plot after learning that shows that now it does not make any mistakes any more.
Then do this again in env3.

Reply:
I think this would make the notebook unnecessarily longer, and we do it already in the regular training. Also we are not yet at a stage where this needs to be a perfect tutorial notebook (we might create something specific later)

Reply:
It's 1 extra line of code and 1 extra cell split, right? (or 2 if done again for env3, although that one I think is less meaningful). I added it when running through the notebook, just to confirm that what I thought was happening, was indeed happening. Even if not a formal and smoothed out tutorial notebook, I think this is a nice starting point for a new user to work from. Why not make it easy for them to see what is changing at first glance?

I also don't understand your argument that it's done already for the regular training. By that logic, none of the outputs need to be shown for the sequential training, as they are already visible for the regular training.

Anyway, I'll approve the PR and you decide whether you want to add this or not.